package.json

```diff
@@ -36,7 +36,8 @@
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0",
     "reselect": "^3.0.1",
-    "sizeof": "^1.0.0"
+    "sizeof": "^1.0.0",
+    "superagent": "^3.8.3"
```

Comment:
This looks like a bad merge. (I nixed `superagent` Friday.) Did you add code that requires `superagent`? If not, please `npm remove` it.